From ebfbf0c5320a5d88aa04e78c3da8870f368305b9 Mon Sep 17 00:00:00 2001
From: FunnyBiu <549308442@qq.com>
Date: Tue, 21 Mar 2023 16:30:52 +0800
Subject: [PATCH] Update basename.c

---
 mingw-w64-crt/misc/basename.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mingw-w64-crt/misc/basename.c b/mingw-w64-crt/misc/basename.c
index c45dbbb36..a19137116 100644
--- a/mingw-w64-crt/misc/basename.c
+++ b/mingw-w64-crt/misc/basename.c
@@ -41,7 +41,7 @@ basename (char *path)
 
   if (locale != NULL)
     locale = strdup (locale);
-  setlocale (LC_CTYPE, "");
+  setlocale (LC_CTYPE, 0);
 
   if (path && *path)
     {
