From 7001e4b2635948cc6283f0534afa08457a342fd0 Mon Sep 17 00:00:00 2001
From: FunnyBiu <549308442@qq.com>
Date: Tue, 21 Mar 2023 16:26:56 +0800
Subject: [PATCH] Create dirname.c

---
 mingw-w64-crt/misc/dirname.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mingw-w64-crt/misc/dirname.c b/mingw-w64-crt/misc/dirname.c
index 9c5cf87db..ce38a3190 100644
--- a/mingw-w64-crt/misc/dirname.c
+++ b/mingw-w64-crt/misc/dirname.c
@@ -40,7 +40,7 @@ dirname(char *path)
 
   if (locale != NULL)
     locale = strdup (locale);
-  setlocale (LC_CTYPE, "");
+  setlocale (LC_CTYPE, 0);
 
   if (path && *path)
     {
